.github/workflows/main.yml

```diff
@@ -163,6 +163,7 @@ jobs:
     - run: |
         rustup target add wasm32-unknown-emscripten
         rustup target add armv7-unknown-linux-gnueabihf
+        sudo apt-get install -y gcc-arm-linux-gnueabihf
```

Comment:
How come this was needed with this change? I would have figured that rsix was already part of wasi-common's dep tree to have this required already...

Reply:
It's actually an unrelated change; a contributor recently added 32-bit ARM support in rsix, so it now needs cross tools to be cross-compiled.

Reply:
How come this is required at `cargo check`-time though? Did rsix previously not have C code to build and now it does?

Reply:
Rsix has two backends. The libc backend and the raw linux syscall backend. The latter requires compiling external assembly and is only used on supported linux targets. Previously 32-bit ARM was not one of such targets and thus would use the libc backend AFAIU.